### ISSUE TITLE
Any plans to make dnsproxy available as a service?

### ISSUE THREAD

**Issue by gspannu** [AdguardTeam/dnsproxy#209]
### Are there any plans to make dnsproxy available as a service?

_Something similar to AdGuardHome..._

I can configure AdGuardHome to run as a service on my ASUS Merlin Router (with Entware).

**Example of AdGuard Home**

1) Contents of _/opt/etc/init.d/_
```
#!/bin/sh
ENABLED=yes
WORK_DIR=/opt/etc/AdGuardHome
PROCS=AdGuardHome
ARGS="-c $WORK_DIR/AdGuardHome.yaml -w "$WORK_DIR" --no-check-update -l syslog"
PREARGS="nohup"
PRECMD="killall -9 dnsmasq"
POSTCMD="service restart_dnsmasq"
DESC=$PROCS
PATH=/opt/sbin:/opt/bin:/opt/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
export TZ=$(cat /etc/TZ)
. /opt/etc/init.d/rc.func
```
2) And then I all I need to do is make changes to my _dnsmasq conf_
```
port=553
local=/1.168.192.in-addr.arpa/
```
This makes AdGuardHome as the DNS server running on port 53 and my existing dnsmasq still operating on my router (on custom port 553 and only for local queries)

---
### Or some workaround/ hack to utilise it the same way as AdGuardHome ?

What I am looking to achieve is...
- Run dnsproxy as a service and listen on port 53
- dnsproxy will then forward the queries to different upstream **AdGuardHome** resolvers (configured in dnsproxy.yaml)
- existing dnsmasq (of the Asus router) will continue to operate as normal; but listening on port 553 and responding only to local queries.

**Comment by gspannu** [comment]
@ameshkov 
Any plans to make dnsproxy as a service - the same principles as for AdGuardHome.

**Comment by ameshkov** [comment]
There's actually an open issue about that already so this can be closed as a duplicate: #39 

We would like to use the same code as used in AdGuard Home for that, but this requires time and we are focusing on AdGuard Home more.
